extensions/blocks/mailchimp/edit.js

```diff
@@ -149,23 +149,21 @@ class MailchimpSubscribeEdit extends Component {
 			</Placeholder>
 		);
 		const placeholder = (
-			<Placeholder icon={ icon } label={ __( 'Mailchimp', 'jetpack' ) } notices={ notices }>
-				<div className="components-placeholder__instructions">
-					{ __(
-						'You need to connect your Mailchimp account and choose a list in order to start collecting Email subscribers.',
-						'jetpack'
-					) }
-					<br />
-					<br />
-					<Button isDefault isLarge href={ connectURL } target="_blank">
-						{ __( 'Set up Mailchimp form', 'jetpack' ) }
-					</Button>
-					<br />
```

Comment:
I think the line breaks were needed in some scenarios. In the example below, I am on a site that does not use the Gutenberg plugin:

![image](https://user-images.githubusercontent.com/426388/72548796-bcf6ef80-388f-11ea-95e4-1ff08dba6deb.png)

vs. before the patch:

![image](https://user-images.githubusercontent.com/426388/72548810-c718ee00-388f-11ea-9532-e648f240ccf5.png)

Reply:
Updated in https://github.com/Automattic/jetpack/pull/14371#issuecomment-575345537 👍